src/land/owp_snow_obs.py

```diff
@@ -32,11 +32,11 @@
     to IODA output files. """)
 
 # obs file name -> ioda file name
-output_var_dict = {'snow_depth_m': 'snow_depth', 'snow_water_equivalent_mm': 'swe'}
+output_var_dict = {'snow_depth_m': 'totalSnowDepth', 'snow_water_equivalent_mm': 'swe'}
```

Comment:
Presumably `swe` also needs updated?

Reply:
swe -> snowWaterEquivalent, units is changed: mm -> kg m-2